README.md

```diff
@@ -13,6 +13,18 @@ Control plane clusters alert rules are maintained by the same team, but are kept
 different
 [repository](https://gitlab.cee.redhat.com/service/app-interface/-/tree/master/resources/stonesoup/argocd-control-plane/monitoring).
 
+The different alerting rules in this repository are:
+
+* [**Alert Rule ControllerReconciliationErrors**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-controllerReconciliationErrors.md?ref_type=heads)
+
+* [**Alert Rule CrashLoopBackOff**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-crashLoopBackOff.md?ref_type=heads)
+
+* [**Alert Rule PodsNotReady**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-PodsNotReady.md?ref_type=heads)
+
+* [**Alert rule for PersistentVolumeIssues**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-pesistentVolumeIssues.md?ref_type=heads)
+
+* [**Alert Rule QuotaExceeded**]()
```

Comment:
There's no link here

Reply:
@yftacherzog  The alerts for the control plane and data plane have just been updated. I couldn't find a link to that one initially, but I have updated it.